code/modules/admin/antag_panel.dm

```diff
@@ -165,10 +165,8 @@ GLOBAL_VAR(antag_prototypes)
 					continue
 				pref_source = prototype
 				break
-		if(pref_source.job_rank)
-			if(!is_banned_from(src.key, pref_source.job_rank))
-				antag_header_parts += pref_source.enabled_in_preferences(src) ? "Enabled in Prefs" : "Disabled in Prefs"
-			else
+		if(pref_source.banning_key)
+			if(is_banned_from(src.key, pref_source.banning_key))
 				antag_header_parts += "<span class='bad'><b>\[BANNED\]</b></span>"
```

Comment:
Why remove the ability to see if someone has it enabled in prefs? This was specifically added because players frequently claim they disabled a role when it was enabled

Reply:
Because the relationship between preferences and banning keys is not 1<->1. There are two preferences for wizard, for example. Which would be shown? It's not very intuitive

Reply:
Re-added, now showing any related preference toggles for the antag datum